MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationComponent.java

```diff
@@ -1014,20 +1045,62 @@ public void forceLocationUpdate(@Nullable Location location) {
    * @param lookAheadUpdate If set to true, the last location's timestamp has to be greater than current timestamp and
    *                        should represent the time at which the animation should actually reach this position,
    *                        cutting out the time interpolation delay.
+   * @deprecated use {@link #forceLocationUpdate(LocationUpdate)} instead
    */
+  @Deprecated
```

Comment:
Perhaps do not see whole picture but why is logic changed for already deprecated method?
IMHO it should have worked the same way it did before while new non-deprecated method is introduced.

Reply:
I moved the calculation of duration out of the animator coordinator to maintain compatibility since the coordinator now accepts the duration as an argument.